src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <div class="main">
+        <h1>Leaderboard</h1>
+        <section class="main-sec">
+          <div class="recent-scores">
+            <div class="recent-scores-top">
+              <h3>Recent scores</h3>
+              <button id="refresh-btn">Refresh</button>
+            </div>
+            <div class="recent-scores-bottom">
+              <ul class="score-list">
+               
+              </ul>
+            </div>
+          </div>
+          <div class="add-scores">
+            <div class="add-score-title">
+              <h3>Add your score</h3>
+            </div>
+            <div class="add-score-form">
+              <form>
+                <input id="input-name" type="text" placeholder="Your name">
+                <input id="input-score" type="text" placeholder="Your score">
```

Comment:
## Optional Change
- [ ]  it would be nice if you could make this input field accept only numbers instead of text. As it's one can enter a text as a score but I believe the score is supposed to be a number. Kindly make the type of this input field be a number.

**See Input Text**

![image](https://user-images.githubusercontent.com/50941074/181859971-a9ee53ea-35cf-4d5b-811e-255ba754d473.png)